code/modules/multiz/structures.dm

```diff
@@ -205,28 +200,14 @@
 	allowed_directions = UP|DOWN
 	icon_state = "ladder11"
 
-
-
-
-
-
-
-
-
-
 /obj/structure/stairs
-	name = "Stairs"
+	name = "stairs"
 	desc = "Stairs leading to another deck.  Not too useful if the gravity goes out."
 	icon = 'icons/obj/stairs.dmi'
-	breakable = FALSE
```

Comment:
stears being unbreakable was a design decision, put that back in

Reply:
> stairs being unbreakable was a design decision, put that back in

ready